src/odb/src/db/dbPowerSwitch.cpp

```diff
@@ -118,6 +121,9 @@ dbIStream& operator>>(dbIStream& stream, _dbPowerSwitch& obj)
   stream >> obj._control_port;
   stream >> obj._on_state;
   stream >> obj._control_net;
+  stream >> obj._lib_cell;
+  stream >> obj._lib;
+  stream >> obj._port_map;
```

Comment:
You need to provide backward compatibility here (load is conditional db_schema_upf_power_switch_mapping)

Reply:
done